fontforgeexe/savefontdlg.c

```diff
@@ -87,6 +87,7 @@ static int nfnt_warned = false, post_warned = false;
 #define CID_TTF_OldKern		1109
 #define CID_TTF_GlyphMap	1110
 #define CID_TTF_OFM		1111
+#define CID_TTF_NoMacNames	1112
```

Comment:
Are you sure we're not stepping on any phantom toes here?

Reply:
Of course not, but I tested it and it worked.

Reply:
Okay. I guess since the value doesn't end up in SFD, we can always renumber it if we find a problem later.